jquery.pjax.js

```diff
@@ -576,7 +576,6 @@ function stripParam(url, name) {
   return url
     .replace(new RegExp('[?&]' + name + '=[^&#]*'), '')
     .replace(/[?&]($|#)/, '\1')
-    .replace(/[?&]/, '?')
```

Comment:
In what cases do the _ or _pjax parameters end up first in query string? I assumed they would always be appended.